sdk/react/panels/cast/CastConnectedScreen.js

```diff
@@ -0,0 +1,345 @@
+        isShow={isShown}
+        screenReaderEnabled={screenReaderEnabled}
+        stereoSupported={stereoSupported}
+        config={{
+          controlBar,
+          castControls,
+          buttons,
+          icons,
+          live,
+          general,
+          selectedPlaybackSpeedRate,
+        }}
+        inCastMode={inCastMode}
+      />
+    );
+  }
+
+  renderCastPlayPause() {
+    const { props } = this;
+    const {
+      width, height, config, live, playhead, duration, rate, playing, loading,
+    } = props;
+    const {
+      play, previous, next, pause, forward, replay,
+    } = config.icons;
+
+    const iconFontSize = ResponsiveDesignManager.makeResponsiveMultiplier(width, UI_SIZES.VIDEOVIEW_PLAYPAUSE);
+    const seekVisible = !config.live.forceDvrDisabled || !live;
+    const notInLiveRegion = playhead <= duration * VALUES.LIVE_THRESHOLD;
+    const icons = {
+      play: {
+        icon: play.fontString,
+        fontFamily: play.fontFamilyName,
+      },
+      previous: {
+        icon: previous.fontString,
+        fontFamily: previous.fontFamilyName,
+      },
+      next: {
+        icon: next.fontString,
+        fontFamily: next.fontFamilyName,
+      },
+      pause: {
+        icon: pause.fontString,
+        fontFamily: pause.fontFamilyName,
+      },
+      seekForward: {
+        icon: forward.fontString,
+        fontFamily: forward.fontFamilyName,
+      },
+      seekBackward: {
+        icon: replay.fontString,
+        fontFamily: replay.fontFamilyName,
+      },
+    };
+
+    const showButtons = true;
+    const showSeekButtons = true;
+
+    return (
+      <CastPlayPauseButtons
+        icons={icons}
+        seekEnabled={seekVisible}
+        config={config}
+        ffActive={live ? notInLiveRegion : true}
+        onPress={name => this.handlePress(name)}
+        onSeekPressed={isForward => this.onSeekPressed(isForward)}
+        onSwitchPressed={isForward => this.onSwitchPressed(isForward)}
+        seekForwardValue={config.castControls.skipForwardTime}
+        seekBackwardValue={config.castControls.skipBackwardTime}
+        frameWidth={width}
+        frameHeight={height}
+        buttonWidth={iconFontSize}
+        buttonHeight={iconFontSize}
+        fontSize={iconFontSize}
+        showButton={showButtons}
+        isLive={live}
+        showSeekButtons={showSeekButtons}
+        rate={rate}
+        playing={playing}
+        loading={loading}
+      />
+    );
+  }
+
+  render() {
+    const { height, width } = this.props;
+
+    return (
+      <View
+        accessible={false}
+        style={[styles.castConnectedScreen, {
+          width,
+          height,
+        }]}
+      >
+        {this.renderTopPanel()}
+        {CastConnectedScreen.renderBorder()}
+        {this.renderPlaceholder()}
+        {this.renderCastPlayPause()}
+        {this.renderBottomOverlay()}
+      </View>
+    );
+  }
+}
+
+module.exports = CastConnectedScreen;
```

Comment:
Use `export default class` over `module.exports`

Reply:
We will fix it during the process of lint errors fixing.